examples/flux/sampling.py

```diff
@@ -0,0 +1,156 @@
+import math
+from typing import Callable, Dict, List, Union
+
+import numpy as np
+
+import mindspore as ms
+from mindspore import Tensor, ops
+
+from .model import Flux
+from .modules.conditioner import HFEmbedder
+from .util import is_torch_available
+
+if is_torch_available():
+    import torch
+
+
+def get_noise(
+    num_samples: int,
+    height: int,
+    width: int,
+    dtype: ms.Type,
+    seed: int,
+):
+    shape = (
+        num_samples,
+        16,
+        # allow for packing
+        2 * math.ceil(height / 16),
+        2 * math.ceil(width / 16),
+    )
+
+    if is_torch_available():
```

Comment:
consider remove the torch branch after precision is checked

Reply:
IMO generating tensor from torch RNG if torch is available otherwise using numpy RNG does not result in much overhead, i think making the generated images keep consistent with torch not only in precision validation but also in daily usage is not a bad idea? wdyt here?